index.html

```diff
@@ -6,11 +6,94 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
     <!-- Add a link to your css file here -->
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <main>
+    <header class= "header">
+            <div>
+                <img class="logo" src="images/cake-logo.png" alt="Logo Image" >
+            </div>
+            <div>
+                <i class="fa fa-envelope"></i>
+            </div>
+    </header>
+    <nav>  
+        <div class="topnav">
```

Comment:
`<ul>`s and `<li>`s (short for ***u**nordered **l**ist*, and ***l**ist **i**tem*) are great semantic tags for most lists, including list of links in navigation bars, e.g.:

```html
<nav>
  <ul class="topnav">
    <li><a href="#">Home</a></li>
    <li><a href="#">Cakes</a></li>
    ...
  </ul>
</nav>
```